core/extraction/common.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
```

Comment:
Just wondering, does it make sense to use this opportunity to move ALL common extraction methods from Round 3 to here? Since Round 4 imports a lot of R3 methods, but really lots of them are round agnostic? I think we chatted about this before in a similar bit of work, but it might make more sense to do it now?

Reply:
Yeah I see what you mean.. They're actually all common in that sense. The only difference is the top level function that calls them with the particular parameters... I think this should be a separate ticket to be honest.